src/ZipkinExporter.php

```diff
@@ -55,25 +58,45 @@ class ZipkinExporter implements ExporterInterface
      */
     private $localEndpoint;
 
+    /**
+    * @var array
+    */
+    private $defaultAttributes;
+
     /**
      * Create a new ZipkinExporter
      *
      * @param string $name The name of this application
+     * @param string $accessToken The access token for LightStep
      * @param string $endpointUrl (optional) The url for the span reporting
      *        endpoint. **Defaults to** `http://localhost:9411/api/v2/spans`
+     * @param array $defaultAttributes(optional) A set of of default tags to be
+     *        applied to all spans.
      * @param array $server (optional) The server array to search for the
      *        SERVER_PORT. **Defaults to** $_SERVER
      */
-    public function __construct($name, $endpointUrl = null, array $server = null)
+    public function __construct($name, $accessToken, $endpointUrl = null, array $defaultAttributes= null, array $server = null)
     {
         $server = $server ?: $_SERVER;
+        $this->accessToken = $accessToken;
         $this->endpointUrl = ($endpointUrl === null) ? self::DEFAULT_ENDPOINT : $endpointUrl;
         $this->localEndpoint = [
             'serviceName' => $name
         ];
+
         if (array_key_exists('SERVER_PORT', $server)) {
             $this->localEndpoint['port'] = intval($server['SERVER_PORT']);
         }
+
+        $this->defaultAttributes= [
+          LIGHTSTEP_HOSTNAME => gethostname(),
+          LIGHTSTEP_PLATFORM_VERSION => phpversion(),
+          LIGHTSTEP_ACCESS_TOKEN => $accessToken
+        ];
+
+        if (!empty($defaultAttributes)) {
+          $this->defaultAttributes = array_merge($this->defaultAttributes, $defaultAttributes);
```

Comment:
Do we want to allow these LightStep attributes to be overridable by the client? If so, then this ordering of the `array_merge` parameters is correct. If we don't, then we should swap the order so that the ones we set always take precedence.

Reply:
Theoretically yes they should be able to be overridden by what's on the span, similar to how tracer tags can be over ridden

Reply:
yeah, similar to how users can override the tracer tags, and then how span tags override tracer tags.